docs/guides/using-suites.md

```diff
@@ -0,0 +1,98 @@
+# Using suites
+
+With increasing number of MFTF tests, it's important to have a mechanism to organize and consolidate them for ease-of-use.
+
+## What is a suite?
+
+Suite is a collection of MFTF tests that are intended to test specific behaviors of Magento. It may contain common initialization and clean up steps specific to the test cases included. It allows you to include, exclude and/or group tests with preconditions and post conditions.
+You can create a suite referencing tests, test groups and modules.
+
+### How is a suite defined?
+
+Suite must be defined under `<magento 2 root>/dev/tests/acceptance/tests/_suite` as an xml file. The generated tests for each suite are grouped into their own directory under `<magento 2 root>/dev/tests/acceptance/tests/functional/Magento/FunctionalTest/_generated/`
+
+### What is the format of a suite?
+
+A suite comprises of the below blocks:
+
+* `<before>` : executes precondition once per suite run.
+* `<after>`  : executes postcondition once per suite run.
+* `<include>`: includes specific tests/groups/modules in the suite.
+* `<exclude>`: excludes specific tests/groups/modules from the suite.
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+
+<suites xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../dev/tests/acceptance/vendor/magento/magento2-functional-testing-framework/src/Magento/FunctionalTestingFramework/Suite/etc/suiteSchema.xsd">
```

Comment:
For the schema attribute, use the `urn` instead of this relative path.

Reply:
fixed.